### DISCUSSION TITLE
How to get the closest point to the camera from a point cloud?

### DISCUSSION THREAD

**Discussion by Luclaft** [isl-org/Open3D#5811] [Q&A]
### Checklist

- [X] I have searched for [similar issues](https://github.com/isl-org/Open3D/issues).
- [X] For Python issues, I have tested with the [latest development wheel](http://www.open3d.org/docs/latest/getting_started.html#development-version-pip).
- [X] I have checked the [release documentation](http://www.open3d.org/docs/release/) and the [latest documentation](http://www.open3d.org/docs/latest/) (for `master` branch).

### My Question

Hello,

I'm working on an Open3D project with a RealSense camera, Open3D gives me a rendering in real time of the scene that the camera sees with PointCloud.
Now I would like to highlight the nearest point of the points cloud.
Can you help me please?

**Comment by ssheorey** [comment]
I think you are looking for nearest neighbor search.

http://www.open3d.org/docs/latest/python_api/open3d.geometry.KDTreeFlann.html#open3d.geometry.KDTreeFlann.search_knn_vector_3d
